core/lib/constants/src/contracts.rs

```diff
@@ -130,3 +130,11 @@ pub const MINT_AND_BURN_ADDRESS: H160 = H160::zero();
 
 // The `storage_log.value` database value for a contract that was deployed in a failed transaction.
 pub const FAILED_CONTRACT_DEPLOYMENT_BYTECODE_HASH: H256 = H256::zero();
+
+pub const SHARED_BRIDGE_ETHER_TOKEN_ADDRESS: Address = H160([
```

Comment:
please add comment on what this is, and how it differs from L2_ETH_TOKEN_ADDRESS (0x800a)

Reply:
also - AFAIK 0x01 is ecrecover precompile address, right?

Reply:
Yes, cc @kelemeno

Reply:
This is the address used in L1 contracts (such as the shared bridge) for Eth, specifically to mark that a chain has the base token eth. 

 L2_ETH_TOKEN_ADDRESS marks the address of the base token system contract on L2. This will actually be used for every base token, so we cannot reuse that address for this ( that would just cause more confusion)